virtualhosts/etc/e-smith/templates/httpd/vhost-extra/15RhPhpScl_values

```diff
@@ -12,10 +12,18 @@ if ($PhpVersion ne 'default')
     $OUT .= "\n";
     $OUT .= "# use rh-php-fpm with mod_proxy_fcgi by socket 
 # from rh repository  instead of the default PHP\n";
-$OUT .= << "EOF";
+    if ($PhpVersion ne 'php80') {
+        $OUT .= << "EOF";
 <FilesMatch \.php\$>
   SetHandler "proxy:unix:/var/run/rh-$PhpVersion-php-fpm/$VhostName-$PhpVersion.sock|fcgi://localhost"
 </FilesMatch>
 EOF
+        } else {
+        $OUT .= << "EOF";
+<FilesMatch \.php\$>
+  SetHandler "proxy:unix:/var/run/$PhpVersion-php-fpm/$VhostName-$PhpVersion.sock|fcgi://localhost"
```

Comment:
Why not retaining the old `rh-` prefix here? The fragment name is `15RhPhpScl_values`... Apart from the socket path it seems there are no other differences with SCLo pacakges here.

Also the prop name is `PhpRhVersion`. Maybe retaining that "Rh" prefix requires less changes to the existing code.

Reply:
we use the same run directory, no condition inside the template